src/main/python/docs/source/matrix.rst

```diff
@@ -14,17 +14,23 @@
 ..  limitations under the License.
 .. ------------------------------------------------------------------------------
 
+
 Matrix API
 ==========
 
 OperationNode
 -------------
 
+  .. todo
+    The explanation for overloade methods seems weird and does not really describe which
+    methods we mean (magic methods for operators like `+`, `*` etc.).
+    Also I don't understand why that would mean that they return an ``OpeartionNode``.
+
```

Comment:
Some discussion on this would be great @Baunsgaard

Reply:
Probably @sebwrede who commented on this.

Reply:
well call me :telephone_receiver:

Reply:
call you on the red emergency phone for PR comments - nice, good to now ;-).